### PR TITLE
Update doc path

### PR DESCRIPTION
**Pull request by GavinZZ**
### Description of changes
Update doc path as the previous method could be troublesome.

### Checklist

- [ ] Adheres to the [development guidelines](https://github.com/aws/serverless-application-model/blob/develop/DEVELOPMENT_GUIDE.md#development-guidelines)
- [ ] Add/update [transform tests](https://github.com/aws/serverless-application-model/blob/develop/DEVELOPMENT_GUIDE.md#unit-testing-with-multiple-python-versions)
    - [ ] Using correct values
    - [ ] Using wrong values
- [ ] Add/update [integration tests](https://github.com/aws/serverless-application-model/blob/develop/INTEGRATION_TESTS.md)

### Examples?

Please reach out in the comments if you want to add an example. Examples will be 
added to `sam init` through [aws/aws-sam-cli-app-templates](https://github.com/aws/aws-sam-cli-app-templates).

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: samtranslator/internal/schema_source/common.py [hunk 1]
```diff
@@ -32,9 +32,8 @@ class PassThroughProp(pydantic.BaseModel):
 
 LenientBaseModel = pydantic.BaseModel
 
-_packagedir = os.path.dirname(os.path.abspath(__package__))
-_docdir = os.path.join(_packagedir, "schema_source")
-_DOCS = json.loads(Path(_docdir, "docs.json").read_bytes())
+_docdir = os.path.dirname(os.path.abspath(__file__))
+_DOCS = json.loads(Path(_docdir, "sam-docs.json").read_bytes())
```
Comment (ssenchenko) [inline samtranslator/internal/schema_source/common.py:36]: Does it look up the file?

Comment (GavinZZ) [reply samtranslator/internal/schema_source/common.py:36]: Discussed offline. Yes since they're in the same directory.

File: MANIFEST.in [hunk 2]
```diff
@@ -11,5 +11,6 @@ include samtranslator/model/connector_profiles/profiles.json
 include samtranslator/internal/data/aws_managed_policies.json
 include README.md
 include THIRD_PARTY_LICENSES
+include samtranslator/internal/schema_source/sam-docs.json
```
Comment (aahung) [inline MANIFEST.in:14]: Nit: why it is not at line 12

**Review by aahung — APPROVED** [review]

File: MANIFEST.in (hunk 2 above)
Comment (GavinZZ) [reply MANIFEST.in:14]: lol

**Review by xazhao — APPROVED** [review]

**Review by hoffa — APPROVED** [review]

**Pull request merged**